### DISCUSSION TITLE
Every cellphone I have owned - Alan Pope's blog

### DISCUSSION THREAD

**Discussion by giscus** [popey/popey.com-blog#25] [General]
# Every cellphone I have owned - Alan Pope's blog

Time for a listicle!
I’ve felt the need to list (almost) every mobile phone that I can remember owning and using, ever. There’s at least one missing from the list. My stupid old brain won’t let me recall the niche brand of phone I bought some years back. It’ll come to me one day.
There’s a table below with some of the interesting manufacturers specs, and further down are some rambling memories from my experience of owning the various devices.

https://popey.com/blog/2023/09/every-cellphone-i-have-owned/

**Comment by MariusQuabeck** [comment]
I have a Firefox OS phone here that used to be yours :)

**Reply by popey** [reply]
Oh god, I forgot about the Firefox devices. I never used them as daily drivers though, so I don't think they count :)
